src/main/services/user_account/ResetApiCredsService.kt

```diff
@@ -1,2 +1,34 @@
 package main.services.user_account
 
+import kotlinserverless.framework.services.SOAResult
+import kotlinserverless.framework.services.SOAResultType
+import kotlinserverless.framework.services.SOAServiceInterface
+import main.daos.*
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.lang.Exception
+
+/**
+ * This service will be used to reset a user account's ApiCreds
+ */
+object ResetApiCredsService: SOAServiceInterface<ApiCred> {
```

Comment:
So i think we should either return an ApiCredNamespace OR create a new object that is like "NewApiCred" (Similar to NewUserAccount) -- in order to pass back the secret unencrypted